src/main/java/controller/MentorController.java

```diff
@@ -67,8 +66,87 @@ public void handle(HttpExchange httpExchange) throws IOException {
 
 
 
-    private void update() {
+    private void update(HttpExchange httpExchange) throws UnsupportedEncodingException, IOException {
+        System.out.println("update executed");
+        String uri = httpExchange.getRequestURI().toString();
+        String method = httpExchange.getRequestMethod();
+        int userId = 0;
+
+        if (method.equals("GET")) {
+            String userAgent = httpExchange.getRequestHeaders().getFirst("User-agent");
+            JtwigTemplate template = JtwigTemplate.classpathTemplate("/templates/mentor/updateStudent.twig");
+            JtwigModel model = JtwigModel.newModel();
+
+
+            try {
+                userId = this.getIdFromUri(uri);
+                User user = userDAO.seeProfile(userId);
+                model.with("name", user.getFirstName());
+                model.with("surname", user.getLastName());
+                model.with("login", user.getLogin());
+                model.with("password", user.getPassword());
+                model.with("email", user.getEmail());
+                model.with("adress", user.getAddress());
+                model.with("phone", user.getPhoneNum());
+            }catch (DBException exc){
+                System.out.println("DB exception cought in update Student in mentor controller");
```

Comment:
instead of printing some inf about exception its better to print actual exception message (e.printStackTrace() for example)